x/ccv/provider/keeper/misbehaviour.go

```diff
@@ -160,3 +169,14 @@ func (k Keeper) CheckMisbehaviour(ctx sdk.Context, misbehaviour ibctmtypes.Misbe
 
 	return nil
 }
+
+// Check if the given block headers have conflicting state transitions.
+// Note that this method was copied from ConflictingHeaderIsInvalid in CometBFT,
+// see https://github.com/cometbft/cometbft/blob/v0.34.27/types/evidence.go#L285
+func headersStateTransitionsAreConflicting(h1, h2 tmtypes.Header) bool {
```

Comment:
Why not just use the CometBFT method?

Reply:
Bc it'd require creating a `LightClientAttackEvidence` type struct, which can be confusing imo.